### ISSUE TITLE
Error in README guide

### ISSUE THREAD

**Issue by marcospb19** [hhat-lang/hhat_jupyter_kernel#1]
From the README instructions on how to run the kernel:

> You need to clone [H-hat quantum programming language repo](https://github.com/hhat-lang/hhat_lang) and, on the root folder, run the command on terminal (or powershell):
> ```bash
> python3 -m pip install -e .
> ```

When I do this step, I get this error:
```ini
$ python3 -m pip install -e .
Defaulting to user installation because normal site-packages is not writeable
Obtaining file:///home/marcospb19/hhat/lang
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [18 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/home/marcospb19/hhat/lang/setup.py", line 9, in <module>
          setup(
        File "/usr/lib/python3.10/site-packages/setuptools/__init__.py", line 108, in setup
          return distutils.core.setup(**attrs)
        File "/usr/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 147, in setup
          _setup_distribution = dist = klass(attrs)
        File "/usr/lib/python3.10/site-packages/setuptools/dist.py", line 488, in __init__
          _Distribution.__init__(
        File "/usr/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 258, in __init__
          getattr(self.metadata, "set_" + key)(val)
        File "/usr/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 1255, in set_requires
          distutils.versionpredicate.VersionPredicate(v)
        File "/usr/lib/python3.10/site-packages/setuptools/_distutils/versionpredicate.py", line 122, in __init__
          raise ValueError("expected parenthesized list: %r" % paren)
      ValueError: expected parenthesized list: '~=2.0.0'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
```

ping @Doomsk

**Comment by Doomsk** [comment]
now it should work! fixed in the commit [f22db21](https://github.com/hhat-lang/hhat_lang/commit/f22db21028adcd16eed58bf251f8c7ba6f59d7cd).

**Comment by marcospb19** [comment]
Yep, fixed
